client/src/pages/guide/english/cplusplus/stack/index.md

```diff
@@ -163,3 +163,14 @@ int main ()
     2
     1
     Out of loop
+##Uses of Stack
+
+1. Expression Evaluation and Conversion.
+   stacks are used to evaluate and convert expressions like prefx,postfix and infix expression. 
```

Comment:
Can you add whitespace properly after `,`?

Reply:
ok